### DISCUSSION TITLE
`@parcel/css` to provide any workaround for ":where" selector

### DISCUSSION THREAD

**Discussion by northern-mindset** [parcel-bundler/parcel#8262] [Ideas]
Currently global coverage is no more than 90%: [caniuse.com/?search=where](https://caniuse.com/?search=where)

The main feature of `:where` is to achieve 0 specificity which can't be emulated with plain CSS (seemingly). However, maybe emulating `:where` with increased specificity still could be beneficial? At least there may be a chance of proper styling in the old browsers instead of completely unstyled HTML?

Examples:

- https://github.com/tailwindlabs/tailwindcss/discussions/8001
- https://github.com/facebook/docusaurus/issues/7617

`postcss-preset-env` also does not provide any workaround for this selector: https://github.com/csstools/postcss-plugins/discussions/509

**Comment by mischnic** [comment]
Please open an issue here instead: https://github.com/parcel-bundler/parcel-css/issues
